packages/cms-editor/src/main/control/language-tools/translation-wizard/TranslationWizardReview.tsx

```diff
@@ -87,21 +117,24 @@ type TranslationWizardContentProps = {
   translationRequest: CmsTranslationRequest;
 };
 
-const TranslationWizardReviewContent = ({ closeTranslationWizard, translationRequest }: TranslationWizardContentProps) => {
+type TranslationWizardReviewContentProps = {
+  closeTranslationWizard: () => void;
+  translationRequest: CmsTranslationRequest;
+  data: CmsStringDataObject[];
```

Comment:
```suggestion
  data: Array<CmsStringDataObject>;
```
Remember to prefer `Array<>` over `[]`.